utils/similarity/plot.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import sys
+import traceback
+import operator
+import numpy as np
+import random
+
+try:
+    from sklearn import decomposition
+    import matplotlib.pyplot as plt
+except ImportError:
+    decomposition = None
+    plt = None
+    
+from fastText import load_model
+from .cache import load_cache
+
+logger = logging.getLogger("Slither-simil")
+
+def plot(args):
+
+    if decomposition is None or plt is None:
```

Comment:
Consider moving this to the try /except block above

Reply:
If I do that, `matplotlib` won't be an optional dependency anymore. So `slither-simil` will refuse to run without `matplotlib` regardless of the user using the plot mode or not.